### ISSUE TITLE
jobs.select() doesn't work for application, backend

### ISSUE THREAD

**Issue by drmarkwslater** [ganga-devs/ganga#76]
From 6.1.4, you can no longer do:

```
jobs.select(application='Executable')
jobs.select(backend='Local')
```

This should be fixed :)

**Comment by rob-c** [comment]
@drmarkwslater  This might be fixed in the latest 6.1.14 pre (more specifically https://github.com/ganga-devs/ganga/commit/c9c28a55da54d26e57632ed29cfc369353012842)

I had a stab at this this-morning as a short break from something I wasn't able to understand (it helped me solve my original completely unrelated problem as well as it gave me some ideas of what could be broken for another unittest).

**Comment by drmarkwslater** [comment]
Yep - this does indeed seem to be fixed. Thanks Rob!
